static/RELEASE_NOTES.md

```diff
@@ -11,6 +11,9 @@ Here you can find description of changes we've built into each release. While we
 - Add LoadBalancer information to Ingress view
 - Move tracker to an extension
 - Add support page (as an extension)
+- Status bar visual fixes
+- Fix proxy upgrade socket timeouts
+- Fix UI staleness after network issues
 
 ## 3.6.6
```

Comment:
did we lose 3.6.7 notes along the way? Maybe we didn't have any...

Reply:
Those are probably in a v3.6 branch (never synced back to master) /cc @nevalla